pest/src/error.rs

```diff
@@ -173,11 +173,14 @@ impl<R: RuleType> Error<R> {
         self
     }
 
-    /// Renames all `Rule`s from a `ParsingError` `variant`. It does nothing when called on
-    /// `CustomError` `varriant`.
+    /// Renames all `Rule`s if this is a [`ParsingError`]. It does nothing when called on a
+    /// [`CustomError`].
     ///
     /// Useful in order to rename verbose rules or have detailed per-`Rule` formatting.
     ///
+    /// [`ParsingError`]: enum.ErrorVariant.html#variant.ParsingError
```

Comment:
I wonder if intra-rustdoc links were stabilized?

Reply:
I believe you can link between symbols in scope with ``[`symbol`]`` barring a few strange corner cases.

Reply:
That's what I thought, but I can't make it work on my machine. Maybe some part of my toolchain is outdated?

Reply:
Nope, looks like it works on nightly only. Guess we're stuck with explicit URLs for now.